Makefile

```diff
@@ -343,7 +343,7 @@ tar: $(TARBALL)
 $(BINARYTAR): release-only
 	rm -rf $(BINARYNAME)
 	rm -rf out/deps out/Release
-	$(PYTHON) ./configure --prefix=/ --download=all --with-intl=small-icu \
+	$(PYTHON) ./configure --prefix=/ --download=all --with-intl=none \
```

Comment:
I know this is closed, but why `none` out of curiosity? (saves only 4Mb, opposite from how Node binaries ship, turns off Internationalization API…)